CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 
-## Unreleased
+## v0.0.4 Mar 14, 2016
+- Fix #101: vagrant-service-manager version 0.0.4 release @navidshaikh
+- Fix #87 #83: Supports starting OpenShift service as part of config @budhrg @bexelbie @navidshaikh
+- Fix #95: Update hook code to call other middleware first @bexelbie
 - Fix #94: Do not exit if box is not supported @navidshaikh
 - Fixed missing word for plugin installation in README @budhrg
 - Fix links, typos, formatting in CONTRIBUTING.md @budhrg
```

Comment:
@LalatenduMohanty : I apologize, I should have cross checked it earlier. Will update the PR